src/sentry/tagstore/v2/models/tagvalue.py

```diff
@@ -100,12 +101,24 @@ def get_or_create(cls, project_id, _key_id, value, **kwargs):
         rv = cache.get(cache_key)
         created = False
         if rv is None:
-            rv, created = cls.objects.get_or_create(
-                project_id=project_id,
-                _key_id=_key_id,
-                value=value,
-                **kwargs
-            )
+            try:
+                rv, created = cls.objects.get_or_create(
+                    project_id=project_id,
+                    _key_id=_key_id,
+                    value=value,
+                    **kwargs
+                )
+            except MultipleObjectsReturned:
+                # HACK: Temporary fix to deal with non-uniqueness of
+                # rows that should be unique after glibc upgrade changed
+                # how unicode values sort
```

Comment:
Is the code hitting a case where different unicode code points are being treated as the same 'value'?

Reply:
Naw, it's worse than that. The unique constraint isn't doing its job and so actual dupes are being created. https://postgresql.verite.pro/blog/2018/08/27/glibc-upgrade.html

This is to serve a few less 500s while we repair indexes and fix the underlying issue.